src/testUtils/mocks/dataSources.ts

```diff
@@ -3487,3 +3489,54 @@ export const createMockSignedBlock = (
     !signedBlock
   ) as SignedBlock;
 };
+
+/**
+ * @hidden
+ * NOTE: `isEmpty` will be set to true if no value is passed
+ */
+export const createMockProposalData = (proposalData?: {
+  args: unknown[];
+  method: string;
+  section: string;
+}): ProposalData => {
+  const { args, method, section } = proposalData || {
+    args: [],
+    method: '',
+    section: '',
+  };
+
+  return createMockCodec(
+    {
+      args: createMockCodec(args, false),
+      method,
+      section,
+    },
+    !proposalData
+  ) as ProposalData;
+};
+
+export const createMockProposalDetails = (proposalDetails?: {
+  approvals: string;
+  rejections: string;
+  status: string;
+  expiry: Option<Moment> | null;
+  autoClose: boolean;
```

Comment:
Please use the proper types for the arguments. Look at `createMockAuthorization` for guidance (note that options must be passed as options)